### ISSUE TITLE
Add homepage with top tasks

### ISSUE THREAD

**Issue by hidde** [act-rules/act-rules-web#217]
To help people orient themselves better on what they can do on the ACT Rules community website, I propose we add a homepage on `/`, that has a number of tiles that introduce the most important tasks on the website and link to the relevant sections.

The following tasks would be included:

- look at rules
- provide feedback on rules
- find out about developing new rules
- find out where rules are implemented

Each would have a title, perhaps a short introduction, maybe even an icon, and a link to an existing page.

**Comment by nitedog** [comment]
I agree with these core tasks but not sure which should be where on the site. For example, I don't think somebody coming to the homepage would want to directly provide feedback on rules; they would probably want to browse through rules or go to a specific rule first, then from there they might want to provide feedback. I think we need to think more about the typical user journeys through the website.
